packages/playwright-core/src/utils/processLauncher.ts

```diff
@@ -190,14 +190,23 @@ export async function launchProcess(options: LaunchProcessOptions): Promise<Laun
     }
   }
 
+  function sleep(ms: Number) {
+    childProcess.spawnSync(process.execPath, ['-e', 'setTimeout(function(){},' + ms + ')']);
+  }
+
   function killProcessAndCleanup() {
     killProcess();
     options.log(`[pid=${spawnedProcess.pid || 'N/A'}] starting temporary directories cleanup`);
     for (const dir of options.tempDirectories) {
-      try {
-        rimraf.sync(dir, { maxBusyTries: 10 });
-      } catch (e) {
-        options.log(`[pid=${spawnedProcess.pid || 'N/A'}] exception while removing ${dir}: ${e}`);
+      for (let i = 0; i < 10; i++) {
```

Comment:
@liuxingbaoyu Could you please `console.log(new Error().stack)` here to see which specific codepath triggers this sync removal for you? I'd like to better understand the situation before landing a fix.

Reply:
```
    at process.emit (node:events:394:28)    at processEmit [as emit] (F:\playwright\node_modules\signal-exit\index.js:155:35)
    at process.exit (node:internal/process/per_thread:184:15)
    at logErrorAndExit (F:\playwright\packages\playwright-core\lib\cli\cli.js:590:11)
```

Reply:
Thank you, this is interesting! What did you do to trigger an error in CLI? Any specific command that you run?

Reply:
Just use codegen or trace-viewer normally